components/radio-button/demo/demo.html

```diff
@@ -1,47 +1,86 @@
 <!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-	<link rel="stylesheet" href="../coherent-gameface-components-theme.css">
-	<link rel="stylesheet" href="../style.css">
-	<style>
-		body {
-			width: 100vw;
-			height: 100vh;
-			justify-content: center;
-			background-color: rgba(0, 0, 0, 0.5);
-			color: #fff;
-		}
-
-		.section {
-			margin-bottom: 15px;
-		}
-
-		.section-label {
-			font-size: 24px;
-			margin-bottom: 5px;
-		}
-	</style>
+    <link rel="stylesheet" href="../coherent-gameface-components-theme.css">
+    <link rel="stylesheet" href="../style.css">
+    <style>
+        body {
+            width: 100vw;
+            height: 100vh;
+            justify-content: center;
+            background-color: rgba(0, 0, 0, 0.5);
+            color: #fff;
+        }
+
+        .section {
+            margin-bottom: 15px;
+        }
+
+        .section-label {
+            font-size: 24px;
+            margin-bottom: 5px;
+        }
+
+        .custom-buttons [role="radio"] .before,
+        .custom-buttons [role="radio"] .after,
+        .custom-buttons [role="radio"][aria-checked="true"] .after {
+            display: none;
+        }

```

Comment:
Maybe a good enhancement here is to enable users to config radio buttons to show/hide default controls automatically instead of writing CSS as you did here.
For example we can expose `controls-disabled` attribute to the radio button.
```
<radio-button controls-disabled>
    <div class='before' />
    <div class='after' />
</radio-button>
```
and then
```
radio-button[controls-disabled] .before,
radio-button[controls-disabled] .after {
    display: none;
}
```

Reply:
Great idea - added it.